IQSS-Dataverse-Develop.xml

```diff
@@ -78,8 +79,10 @@ mvn surefire-report:report</command>
                 <command>curl -O https://raw.githubusercontent.com/GlobalDataverseCommunityConsortium/dataverse-ansible/master/ec2/ec2-create-instance.sh
 chmod 755 ec2-create-instance.sh
 curl -o jenkins.yml https://raw.githubusercontent.com/GlobalDataverseCommunityConsortium/dataverse-ansible/master/tests/group_vars/jenkins.yml</command>
+                <configuredLocalRules/>
               </hudson.tasks.Shell>
             </buildSteps>
+            <stopOnFailure>false</stopOnFailure>
```

Comment:
Our of curiosity, was `stopOnFailure` unset but false already? And here's we're explicitly setting it to false? If we don't know, that's totally fine. Whatever works to fail the build as appropriate.

Reply:
dunno. we've upgraded jenkins and its plugins several times since the last PR, so this is likely a new setting. if we need to set it to True we can.